fvm/errors/codes.go

```diff
@@ -38,24 +38,24 @@ const (
 	ErrCodeInvalidEnvelopeSignatureError ErrorCode = 1009
 
 	// base errors 1050 - 1100
-	ErrCodeFVMInternalError            ErrorCode = 1050
-	ErrCodeValueError                  ErrorCode = 1051
-	ErrCodeInvalidArgumentError        ErrorCode = 1052
-	ErrCodeInvalidAddressError         ErrorCode = 1053
-	ErrCodeInvalidLocationError        ErrorCode = 1054
-	ErrCodeAccountAuthorizationError   ErrorCode = 1055
-	ErrCodeOperationAuthorizationError ErrorCode = 1056
-	ErrCodeOperationNotSupportedError  ErrorCode = 1057
+	ErrCodeFVMInternalError          ErrorCode = 1050
+	ErrCodeValueError                ErrorCode = 1051
+	ErrCodeInvalidArgumentError      ErrorCode = 1052
+	ErrCodeInvalidAddressError       ErrorCode = 1053
+	ErrCodeInvalidLocationError      ErrorCode = 1054
+	ErrCodeAccountAuthorizationError ErrorCode = 1055
+	// ErrCodeOperationAuthorizationError ErrorCode = 1056
```

Comment:
I think this should be in use, if its not probably I made a mistake somewhere, it should be used to account creation or other limited operations.

Reply:
I tried to find a references in past commits, but I cant find any. Do you have any more clues?

Reply:
this part has to change https://github.com/onflow/flow-go/blob/master/fvm/errors/base.go#L165